### PR TITLE
Retrieve absolute file paths for assets (Windows only).

### PR DESCRIPTION
**Pull request by smokyturbo**

### CODE REVIEW

File: src/main/view/StyleManager.cpp [hunk 1]
```diff
@@ -7,9 +7,42 @@
...
+#if defined(_WIN64) || defined(_WIN32)
+//Windows specific code
+#include <windows.h>
+//End Windows specific code	  
+#endif
+
 using namespace gs;
 
-const std::string StyleManager::assetPath = "assets/";
+const std::string StyleManager::assetPath = "/assets/";
```
Comment (wpoynter) [inline src/main/view/StyleManager.cpp]: Adding the leading forward slash causes quite a bit of grief on Unix

File: src/main/view/StyleManager.cpp [hunk 2]
```diff
@@ -7,9 +7,42 @@
...
+
+	#if defined(_WIN64) || defined(_WIN32)
+		HMODULE hModule = GetModuleHandleW(NULL);
+		WCHAR path[MAX_PATH];
+		int lengthWcharPath = GetModuleFileNameW(hModule, path, MAX_PATH);
+		int lengthStringPath = WideCharToMultiByte(CP_UTF8, 0, path, lengthWcharPath, NULL, 0, NULL, NULL);
+		execPath = std::string(lengthStringPath, 0);
+		WideCharToMultiByte(CP_UTF8, 0, path, lengthWcharPath, &execPath[0], lengthStringPath, NULL, NULL);
+		std::replace(execPath.begin(), execPath.end(), '\\', '/');
+		execPath = execPath.substr(0, execPath.find_last_of("/"));
```
Comment (wpoynter) [inline src/main/view/StyleManager.cpp]: Does this line just remove the last forward slash and everything after it? If so, could we retain the final forward slash and just lose everything after it. Then we won't need the leading forward slash on the const above.

File: src/main/view/StyleManager.cpp [hunk 3]
```diff
@@ -7,10 +7,43 @@
 
 #include "StyleManager.h"
 
+#include <algorithm>
+
+#if defined(_WIN64) || defined(_WIN32)
+//Windows specific code
```
Comment (jademonkey) [inline src/main/view/StyleManager.cpp]: Not sure we need this comment as the line above clearly reads that the code here will only be for Windows. 
Personally i'd only use comments if the code is not clear. Thoughts?

Comment (wpoynter) [inline src/main/view/StyleManager.cpp]: I think it is generally better to make things overly clear, you never know what we might forget over the years to come.

**Pull request merged**